### ISSUE TITLE
reset_db.sh script should wipe out the 'auth_user' table in the scoreboard database

### ISSUE THREAD

**Issue by k21971** [tnnt-devteam/python-backend#24]
Currently, once the reset_db.sh script is ran, any players that have played previously can no longer log in to the site until their entry is removed from the auth_user table. The script itself is pretty straightforward, but I'm not familiar enough with the migrate.py functions to fix this myself 😐

**Comment by copperwater** [comment]
I don't have the hardfought environment set up with multiple players logging in, so try adding this to reset_db.sh:

```sh
. secrets.sh
./manage.py wipe_db --all # <-- new line
./manage.py migrate scoreboard zero && rm -r scoreboard/migrations/* && \
    ./manage.py makemigrations scoreboard && ./manage.py migrate scoreboard
```

This should delete everything in User as well as Player, so does it fix the issue?

**Comment by k21971** [comment]
Yup this works, thanks. I'll make a commit for this.

**Comment by k21971** [comment]
Closed via 53f62f2.
